packages/dart_frog_auth/README.md

```diff
@@ -7,52 +7,51 @@
 [![style: very good analysis][very_good_analysis_badge]][very_good_analysis_link]
 [![License: MIT][license_badge]][license_link]
 
-Header Authorization support for [Dart Frog][dart_frog_link].
+Header authorization support for [Dart Frog][dart_frog_link].
```

Comment:
I wonder if we should keep it with a capital A here, since usually, http headers starts with a capital letter.

Although one could argue that we don't need to use the "http" syntax here 🤔

Reply:
ohhhh, I see Authorization is a specific concept. Yes, agree, we can keep Authorization capitalized.